### ISSUE TITLE
netlify-cms version 2.10.41 gives error "Excessive alias count indicates a resource exhaustion attack"

### ISSUE THREAD

**Issue by ArthurClemens** [netlify/netlify-cms#3566]
With the update of 2.10.41 the admin is no longer rendered and shows this error instead:
```
Error loading the CMS configuration
Config Errors:
  ReferenceError: Excessive alias count indicates a resource exhaustion attack
Check your config.yml file.
```

We have quite an elaborate config yaml where we use aliases freely. It would be limiting to change aliases to repeated copies of identical elements.

**Comment by erezrokah** [comment]
Thanks for reporting @ArthurClemens, we switched our yaml parser recently to get more control on the output, and the new one has protection against that.
I disabled it in https://github.com/netlify/netlify-cms/pull/3570

**Comment by ArthurClemens** [comment]
Thanks for the rapid response.

**Comment by erezrokah** [comment]
This is now released with `netlify-cms@2.10.42`
